src/components/PluginLabelTagCloud.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {useStaticQuery, graphql, navigate} from 'gatsby';
+import {TagCloud} from 'react-tagcloud';
+import './PluginLabelTagCloud.css';
+
+function PluginLabelTagCloud() {
+
+    const excludedLabelSet = new Set(['adopt-this-plugin','deprecated','must-be-labeled','plugin-test']);
+
+    const allLabels = useStaticQuery(graphql`
+        query {
+            labels: allJenkinsPlugin {
+                nodes {
+                  labels
+                }
+            }
+        }
+    `).labels.nodes.reduce((prev, nodes) => {
+
+        nodes.labels.reduce((prevLabel, obj) => {
+            if(!excludedLabelSet.has(obj)) {
+                prev[obj] = (prev[obj] || 0) + 1;
+            }
+            return obj;
+
+        }, {});
+
+        return prev;
+    }, {});
+
+    const tags = Object.entries(allLabels).map(([value, count]) => {return {value, count};});
```

Comment:
`Object.entries` is not supported on IE https://caniuse.com/object-entries. Maybe `Object.values` would serve you better here but it's also not supported. I think a polyfill is needed or use `for .. in` instead..